pkg/await/apps_deployment.go

```diff
@@ -503,6 +541,31 @@ func (dia *deploymentInitAwaiter) changeTriggeredRollout() bool {
 	return len(fields) > 0
 }
 
+func (dia *deploymentInitAwaiter) checkPersistentVolumeClaimStatus() {
+	inputs := dia.config.currentInputs
+
+	glog.V(3).Infof("Checking PersistentVolumeClaims status for Deployment '%s'", inputs.GetName())
+
+	allPVCsReady := true // bool used to do a rolling AND operation across all PVCs statuses
+	for _, pvc := range dia.pvcs {
+		phase, hasConditions := openapi.Pluck(pvc.Object, "status", "phase")
+		if !hasConditions {
+			return
+		}
+
+		// Success occurs when all PersistentVolumeClaims are marked as Bound
+		if phase == statusBound {
+			allPVCsReady = allPVCsReady && true
```

Comment:
This appears to be a no-op. Should be able to simplify the conditional logic to ignore this branch.